components/gitpod-protocol/src/gitpod-service.ts

```diff
@@ -294,7 +294,12 @@ export interface GitpodServer extends JsonRpcServer<GitpodClient>, AdminServer,
     getSpendingLimitForTeam(teamId: string): Promise<number | undefined>;
     setSpendingLimitForTeam(teamId: string, spendingLimit: number): Promise<void>;
 
-    listBilledUsage(attributionId: string, from?: number, to?: number): Promise<BillableSession[]>;
+    listBilledUsage(
+        attributionId: string,
+        startedTimeOrder: SortOrder,
+        from?: number,
+        to?: number,
```

Comment:
@laushinka, please introduce `ListBilledUsageParams` and wrap the args, for 2 reasons:
1. it's likely this args list may be extended, right?
2. optional args at the end are simply not working with jsonrpc, because behind the scene there will be another param added (I think it's a cancellation token) which leads to nasty bugs in JS evaluation.

Reply:
> optional args at the end are simply not working with jsonrpc, because behind the scene there will be another param added (I think it's a cancellation token) which leads to nasty bugs in JS evaluation.

@AlexTugarev Is there a solution for this or can nothing be optional?